app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<section class="incomplete-task">
+  <h3> Incomplete Tasks</h3>
+  <ul>
+  <% @tasks.each do |task| %>
+    <% if task.complete == false %>
```

Comment:
I'd recommend using these booleans directly rather than checking equality. For example, `if !task.complete` rather than `if task.complete == false`